README.md

```diff
@@ -329,6 +329,7 @@ Table of Contents
 
 ## Data Visualization on Maps
 
+   * http://geocod.io - Geocoding via API or CSV Upload
```

Comment:
can you mention about the first 2500 lookups being free? nice to have a quick glance info about the free tier